app/models/filters/filter_base.rb

```diff
@@ -649,7 +649,8 @@ def cohort_columns_for_select
 
     def cohort_columns_for_dates
       GrdaWarehouse::Cohort.available_columns.select do |column|
-        column.class.ancestors.include?(CohortColumns::CohortDate)
+        # Ignore non-dates and untranslated custom dates
+        column.class.ancestors.include?(CohortColumns::CohortDate) && ! column.title.starts_with?('User')
```

Comment:
Maybe
```suggestion
        column.class.ancestors.include?(CohortColumns::CohortDate) && ! column.title.match(/^User Date [0-9]+$/))
```
to reduce the chance of accidental hiding?

Reply:
Thanks.  Using `.match?(/^User Date \d+$/)`